### PR TITLE
WIP: Enabling SSL

### PR DESCRIPTION
**Pull request by lucperkins**
Attn @macintux

### CODE REVIEW

**Comment by macintux** [comment]
Could you give me an example or two of where you'd like to link this from? That would help me understand its role.

File: source/languages/en/riak/ops/running/authz.md [hunk 1]
```diff
@@ -749,6 +748,37 @@ installing it can be found in [[Installing Erlang]]. This issue should
...
+and port. The following configuration would establish port 8088 on
+`localhost` as the HTTPS port:
+
+```riakconf
+listener.https.$name = 127.0.0.1:8088
+
+# By default, "internal" is used as the "name" setting
+```
+
+```appconfig
```
Comment (macintux) [inline source/languages/en/riak/ops/running/authz.md]: I'm a bit confused as to why the `app.config` settings are shown here. If we really want both, it should be more clear that you only need one of these. (Unless I'm really confused, which is possible.)

Comment (lucperkins) [inline source/languages/en/riak/ops/running/authz.md]: @macintux The `app.config` settings are shown in all of the ops materials for the sake of people upgrading to 2.0 from 1.4.x

**Comment by lucperkins** [comment]
@macintux So, one example is the [Riak Control](http://docs.basho.com/riak/latest/ops/advanced/riak-control/), which I'm currently updating. That's a place where I need to say "make sure that SSL is enabled," but there's currently nowhere I can point to in the docs that actually makes it clear how to do that.

**Comment by macintux** [comment]
Riak Control already has an enabling SSL section.

I'm sorry to be such a pain about this, I'm just struggling to figure out what to include that won't water down or otherwise confuse the very careful checklist. It's easy to completely break a production Riak environment by enabling security, so having a small "enabling SSL" section that confuses that message is problematic.

**Comment by lucperkins** [comment]
@macintux The issue is that I think the Riak Control doc is an awkward place to have our primary SSL documentation, and so that doc is undergoing some [re-org](https://github.com/basho/basho_docs/pull/1497/files).

I think it's important to make it as explicit as possible what it takes to get SSL to work. At the moment, that isn't made clear anywhere, as far as I can tell. As for the checklist, the enabling SSL item will still be there but will link to the section below.

**Pull request merged**